lib/ldclient-ai/ld_ai_client.rb

```diff
@@ -0,0 +1,179 @@
+# frozen_string_literal: true
```

Comment:
I would remove the `ld_ai_` prefixes from this and the `ld_ai_config_tracker.rb` files. Those aren't needed, and are implied since they are under the `ldclient-ai` folder.